qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -114,12 +119,22 @@ def __init__(
         self._backend_name = backend_name
         self._backend_version = backend_version
 
-        if isinstance(library, list):
-            raise NotImplementedError(
-                "Passing a list of libraries from which to instantiate "
-                "will be supported in future releases."
+        if library:
+            warnings.warn(
+                f"library has been deprecated, please provide libraries instead."
+                "This warning will be removed with backport in Qiskit Experiments 0.4.",
```

Comment:
What does "removed with backport" mean?

Reply:
TBH I'm not entirely sure but some hints can be gleaned from Wiki: https://en.wiktionary.org/wiki/backport. This came from one of @nkanazawa1989 PRs that was merged.

Reply:
oh I just meant "old option is implicitly replaced with new option name and set to the instance" by "backport". In the next release we'll remove this warning with corresponding logic (I think this is kind of okey since we are still in the active development phase)

Reply:
Okay, I see what you mean. You are thinking of the "library" argument as being "backported" into version 0.3, but usually backport pulling a bug fix back into an older release. I would say it like this:

```suggestion
                'The "library" argument along with this warning will be removed in Qiskit Experiments 0.4.',
```

Reply:
Thanks, I like this. @eggerdj could you please update original message too?